docs/userguide/dockerizing.md

```diff
@@ -57,6 +57,9 @@ So what happened to our container after that? Well Docker containers
 only run as long as the command you specify is active. Here, as soon as
 `Hello world` was echoed, the container stopped.
 
+  <img src="/userguide/hello_world_images/helloworld.gif">
```

Comment:
Thanks for the feedback.  I went ahead and fixed that and used markdown notation.  Hope it looks good.  =)